CITATION.rst

```diff
@@ -6,6 +6,7 @@ that goes into building and maintaining this project.
 
 If you used RockHound in your research, please consider citing it:
 
-   Uieda, L., and Soler, S.R. (2019). Rockhound: Download geophysical models/datasets
-   and load them in Python (Version 0.1.0). Zenodo. https://doi.org/10.5281/zenodo.3086002
+   Uieda, L., and Soler, S.R., Pesce, A. (2020). Rockhound: Download geophysical
+   models/datasets and load them in Python (Version 0.2.0). Zenodo.
+   https://doi.org/10.5281/zenodo.3627166
```

Comment:
We could replace this part by:

If you used RockHound in your research,  please consider citing the Zenodo archive: https://doi.org/10.5281/zenodo.3086001

This DOI redirects to the latest release at the time. DOIs for specific releases can be found under "Versions" on the Zenodo site. Use one of these if you wish to refer to a specific version.